getting-started/templates/systemlink-secrets.yaml

```diff
@@ -38,6 +38,14 @@ secrets:
     ## This cookie is used to authenticate containers in the RabbitMQ stateful set. Use a strong random sequence of characters.
     ##
     erlangCookie: ""  # <ATTENTION>
+  # Miscellaneous artifact repository credentials.
+  genericArtifactPull:
+    # URL for the repository.
+    url: "" # <ATTENTION>
+    # User for repository access. This information will have been provided with the application.
+    user: "" # <ATTENTION>
```

Comment:
The markdown implies that these values will be the same values set for the image pull secrets. If so, that should be documented here. If possible, we should use anchors to avoid requiring duplicate entries in the file.

Reply:
Done